data/properties/core-downloads.yaml

```diff
@@ -18,60 +21,8 @@ majorVersions:
         link: "https://download-core.sys.truenas.net/11/"
         date: "2019-01-24"
   - lifecycle: "Previous"
-    majorVersion: "12.0"
-    majorLink: "https://download-core.sys.truenas.net/12.0/STABLE/"
-    releases:
-      - name: "12.0-U8.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U8.1/"
-        date: "2022-04-14"
-      - name: "12.0-U8"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U8/"
-        date: "2022-02-02"
-      - name: "12.0-U7"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U7/"
-        date: "2021-12-08"
-      - name: "12.0-U6.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U6.1/"
-        date: "2021-11-11"
-      - name: "12.0-U6"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U6/"
-        date: "2021-10-05"
-      - name: "12.0-U5.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U5.1/"
-        date: "2021-08-17"
-      - name: "12.0-U5"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U5/"
-        date: "2021-08-03"
-      - name: "12.0-U4.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U4.1/"
-        date: "2021-07-13"
-      - name: "12.0-U4"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U4/"
-        date: "2021-06-01"
-      - name: "12.0-U3.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U3.1/"
-        date: "2021-05-04"
-      - name: "12.0-U3"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U3/"
-        date: "2021-04-13"
-      - name: "12.0-U2.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U2.1/"
-        date: "2021-02-23"
-      - name: "12.0-U2"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U2/"
-        date: "2021-02-09"
-      - name: "12.0-U1.1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U1.1/"
-        date: "2021-01-15"
-      - name: "12.0-U1"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/U1/"
-        date: "2020-12-09"
-      - name: "12.0-Release"
-        link: "https://download-core.sys.truenas.net/12.0/STABLE/RELEASE/"
-        date: "2020-10-20"
-  - lifecycle: "Current"
     majorVersion: "13.0"
-    majorLink: "https://download-core.sys.truenas.net/13.0/STABLE/"
+    majorLink: "https://download-core.sys.truenas.net/12.0/STABLE/"
```

Comment:
Why did this change to 12.0 release when it is in listed under the 13.0 version section? Should line 24 be 12.0?

Reply:
Good catch - it's a typo from moving the major versions down (up?) the list and moving 12.0 to the archive.